.github/workflows/container-image.yml

```diff
@@ -118,18 +118,20 @@ jobs:
     strategy:
       matrix:
         os: [centos, fedora, opensuse]
-        arch: [amd64, arm64]
+        arch:
+          - {name: amd64, host: ubuntu-24.04}
+          - {name: arm64, host: ubuntu-24.04-arm}
         exclude:
           - os: centos
-            arch: arm64
-    runs-on: ubuntu-latest
+            arch: {name: arm64, host: ubuntu-24.04-arm}
```

Comment:
`samba-client:default-centos-arm64` is mentioned down the line. Did you forget to remove the `exclude` altogether here?